aula-pratica-02/ex3.c

```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+int n_divisores(int n);
+
+int n_divisores(int n)

```

Comment:
It's never a good thing to abbreviate function names or variable names, as the programs grow in size and complexity, your programs will begin to get more and more complicated to read, if your function names are cristal clear to read out loud, is makes it easier to understand your program, I am used to code like a psycho is reading my code and he knows where I live, and he will get very mad if he don't understands my program :joy: 

```suggestion
int counta_divisores(int n)
```

```suggestion
int count_dividers(int n)
```